pystiche_papers/li_wand_2016/_data.py

```diff
@@ -128,7 +139,7 @@ def images() -> data.DownloadableImageCollection:
             title="Blue Bottle",
             author="Christopher Michel (cmichel67)",
             date="02.09.2014",
-            license=data.NoLicense(),
+            license=license("https://www.flickr.com/photos/cmichel67/15112861945"),
```

Comment:
Are we only using this function once? If so, there is no need for it. Let's just put the message here.